ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -334,4 +334,9 @@ export default defineComponent({
 .error-text {
   font-size: 16px;
 }
+
+.home-location-icon {
```

Comment:
If the icon size is the same across the app, maybe we could put it into the base.scss?

Reply:
good call I'll do that